setup/steps/check_auto_explain_available.go

```diff
@@ -0,0 +1,33 @@
+package steps
+
+import (
+	"errors"
+	"strings"
+
+	s "github.com/pganalyze/collector/setup/state"
+	"github.com/pganalyze/collector/setup/util"
+)
+
+var CheckAutoExplainAvailable = &s.Step{
+	Kind:        s.AutomatedExplainStep,
+	Description: "Prepare for auto_explain install",
```

Comment:
We should update this description (since we don't actually install anything here, we just check things)